asset-transfer-sbe/chaincode-typescript/src/asset.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Object, Property } from 'fabric-contract-api';
+
+@Object()
+export class Asset {
+
+    @Property()
+    public ID: string;
+    
+    @Property()
+    public Value: string;
```

Comment:
Better to make this a number to be consistent with the other samples, and to also show that not all data types besides string are possible.